main.go

```diff
@@ -54,7 +54,6 @@ type configs struct {
 	ICloudContainerEnvironment string `env:"icloud_container_environment"`
 	TeamID                     string `env:"team_id"`
 
-	UseDeprecatedExport               bool   `env:"use_deprecated_export,opt[yes,no]"`
```

Comment:
Please update minimum supported Xcode veresion in line 35.

Reply:
good catch, the new min version: 9